.ci/jenkins_job_build.sh

```diff
@@ -31,6 +35,9 @@ then
 	exit 1
 fi
 
+# If CI running on bare mental, wipe out stale Jenkins WORKSPACE
+[ "${BARE}" == true ] && [ -d "${WORKSPACE}" ] && sudo rm -rf "${WORKSPACE}"
```

Comment:
You can set up the jenkins jobs to do this. You might have to fiddle the perms to work around the tests generating root owned files (like the `sudo` on this `rm` here is doing I think). Previously I have had for instance on my Jenkins jobs:

```bash
#Hack - our scripts leave some things not owned by us....
# so the jenkins user can later nuke the workspace tree
echo "hacking our file ownerships"
sudo chown -R ${USER} ${WORKSPACE}
sudo chgrp -R $(id -ng) ${WORKSPACE}
```

and then tick the (either/both) delete workspace before/after build in the Jenkins job setup dialog.